packages/suite/src/support/tests/hooksHelper.tsx

```diff
@@ -18,9 +18,13 @@ export const renderWithProviders = (store: any, children: React.ReactNode) => {
 };
 
 export const waitForLoader = (text = /Loading/i) => {
-    const loading = screen.queryByText(text);
-    if (loading) {
-        return waitForElementToBeRemoved(() => screen.getByText(text));
+    try {
+        const loading = screen.queryByText(text);
+        if (loading) {
+            return waitForElementToBeRemoved(() => screen.getByText(text));
```

Comment:
also replacing `getByText` with `queryByText` works for me as it's returning  `null` instead of error if match is not found

Reply:
replaced, i will leave try/catch just to be sure